GameOfSwift3/GameOfSwift3/ManagerGame.swift

```diff
@@ -30,9 +33,51 @@ class GameManager {
         playerTwo.resume()
     }
     
+    func startGame() {
+        var attackerPlayer = playerOne
+        var attackedPlayer = playerTwo
+        print("\n🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊")
+        print("🥊🥊🥊 LET THE FIGHT BEGIN !🥊🥊🥊")
+        print("🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊")
+        while playerOne.isTeamAlive() && playerTwo.isTeamAlive() == true {
+            print("\n Use number to choose character for the attack or character you want to treat")
+            attackerPlayer.showTeam()
+            let attacker = attackerPlayer.chooseCharacter()
+            print("\nchoose ennemy to attack or Magus to treat")
+            attackedPlayer.showTeam()
+            let attacked = attackedPlayer.chooseCharacter()
+            fight(attacker: attacker, attacked: attacked)
+            swap(&attackerPlayer, &attackedPlayer)
+            numberOfTurn += 1
+        }
+    }
+    
+    func endOfTheGame() {
+    print("\n🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊")
+    print("🥊🥊🥊🥊🥊🥊 AFTER \(numberOfTurn) ATTACKS 🥊🥊🥊🥊🥊")
+    print("🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊🥊")
+       playerOne.winner()
+       playerTwo.winner()
+        }
+
+    
+    func fight(attacker: Character, attacked: Character) {
+        if attacked.lifePoint > 0 && attacked.lifePoint > attacker.damages() {
+        let damages = attacker.damages()
+        attacked.lifePoint -= damages
+        print("\n🔵🔶🔵🔶🔵🔶🔵🔶🔵🔶🔵🔶🔵🔶🔵🔶🔵🔶🔵🔶🔵🔶🔵🔶🔵🔶🔵🔶")
+        print("🔵🔶\(attacker.name) takes \(damages) points to \(attacked.name) who's got now \(attacked.lifePoint) lifepoints🔵🔶")
+        print("🔵🔶🔵🔶🔵🔶🔵🔶🔵🔶🔵🔶🔵🔶🔵🔶🔵🔶🔵🔶🔵🔶🔵🔶🔵🔶🔵🔶\n")
+        } else {
```

Comment:
If the player choose a dead character the code will go though the else statement but it does nothing else. The problem here is that you will count that as a turn.